poetryOverrides.nix

```diff
@@ -1,7 +1,7 @@
 # Copyright lowRISC Contributors.
 # Licensed under the MIT License, see LICENSE for details.
 # SPDX-License-Identifier: MIT
-{pkgs, ...}: let
+{nixpkgs, ...}: let
```

Comment:
Since this is now only called inside the flake itself, perhaps `...` can be removed?

Reply:
Agreed. will do